src/Spice86.Core/Emulator/InterruptHandlers/Dos/DosInt21Handler.cs

```diff
@@ -127,7 +127,7 @@ private void GetLeadByteTable() {
     /// Reads a character from the standard auxiliary device (usually the keyboard) and stores it in AL.
     /// </summary>
     public void ReadCharacterFromStdAux() {
-        IVirtualDevice? aux = _dos.Devices.Find(x => x is CharacterDevice { Name: "AUX" });
+        IVirtualDevice? aux = _dos.Devices.FirstOrDefault(x => x is CharacterDevice { Name: "AUX" });
```

Comment:
Why change this?

Reply:
Probably an error. The original code is the correct one.

Reply:
Fixed.